helusers/models.py

```diff
@@ -48,6 +48,9 @@ class AbstractUser(DjangoAbstractUser):
     uuid = models.UUIDField(unique=True)
     department_name = models.CharField(max_length=50, null=True, blank=True)
     ad_groups = models.ManyToManyField(ADGroup, blank=True)
+    date_api_token_used = models.DateField(
```

Comment:
I feel something like `api_token_last_used_at` would be more clear, but since it's a date field maybe `api_token_last_used_date`